### PR TITLE
Sean dg edits

### PR DESCRIPTION
**Pull request by SeanHoWB**
editting the DG halfway

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1,14 +1,68 @@
...
+### Purpose
+This document specified architecture and software design decisions for the application, Mindmymoney.
+### Scope
+This describes the software architecture and software design decisions for the implementation
+of Mindmymoney. The intended audience of this document is the developers, designers, and
+software testers of Mindmymoney.
+### Acknowledgements
+We would like to thank [AddressBook-3](https://se-education.org/addressbook-level3/) for assisting us in developing
+Mindmymoney.
+### Using the User Guide
```
Comment (khseah) [inline docs/DeveloperGuide.md:46]:
```suggestion
### Using the Developer Guide
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1,14 +1,68 @@
...
+
+### Purpose
+This document specified architecture and software design decisions for the application, Mindmymoney.
+### Scope
+This describes the software architecture and software design decisions for the implementation
+of Mindmymoney. The intended audience of this document is the developers, designers, and
+software testers of Mindmymoney.
+### Acknowledgements
+We would like to thank [AddressBook-3](https://se-education.org/addressbook-level3/) for assisting us in developing
+Mindmymoney.
```
Comment (khseah) [inline docs/DeveloperGuide.md:45]:
```suggestion
MindMyMoney.
```

File: docs/puml/AddCommandSequenceDiagram.puml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+else else
+AddCommand -> AddCommand: addExpenditure()
+activate AddCommand
+AddCommand --> AddCommand:
+deactivate AddCommand
+AddCommand --> Parser:
+deactivate AddCommand
+end
+
+@enduml
```
Comment (khseah) [inline docs/puml/AddCommandSequenceDiagram.puml:39]: Add new line, github complaining haha

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -1,14 +1,68 @@
...
+> - This tells you that there is some **important** information you should take note of to prevent issues from arising
+    when you are using the application.  
+
+Click on the hyperlinks in the [content page](#Content-Page) to quickly navigate the developer's guide.
 
 ## Design
+### Technologies used
+**MindMyMoney** is written fully in **Java 11** using  Object-Oriented Programming (OOP) paradigm to help structure and
+organise the code. This enables the efficiency of future improvements and revisions.
+Data stored in the application is saved into test files locally on the user's device.
```
Comment (khseah) [inline docs/DeveloperGuide.md:61]: I'm assuming you mean text?
```suggestion
Data stored in the application is saved into text files locally on the user's device.  
```

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -96,16 +150,42 @@ The Storage component:
 
 
 ## Implementation
-This section describes some noteworthy details on how certain features are implemented.
+This section describes some noteworthy details on how certain features of Mindmymoney are implemented.
```
Comment (khseah) [inline docs/DeveloperGuide.md:153]:
```suggestion
This section describes some noteworthy details on how certain features of MindMyMoney are implemented.
```

**Review by khseah — COMMENTED** [review]
Nicely done

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -1,14 +1,68 @@
...
+
+## Introduction
+
+**MindMyMoney** (M<sup>3</sup>) is a desktop app for managing users' personal finances, optimized for use via a
+Command Line Interface (CLI). With the application, users can track their expenses across multiple payment methods,
+calculate monthly expenditure, and set financial goals. The application is targeted at students looking to start
+managing their personal finances.
+
+### Purpose
+This document specified architecture and software design decisions for the application, Mindmymoney.
```
[the remark this answers is not in the document]
Comment (SeanHoWB) [reply docs/DeveloperGuide.md]: resolved

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (SeanHoWB) [reply docs/DeveloperGuide.md:46]: ooopsies, resolved

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (SeanHoWB) [reply docs/DeveloperGuide.md]: resolved for all MMM

File: docs/puml/AddCommandSequenceDiagram.puml (hunk 3 above)
Comment (SeanHoWB) [reply docs/puml/AddCommandSequenceDiagram.puml:39]: dammit git, ok resolved

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -1,14 +1,68 @@
 # Developer Guide
 
-## Acknowledgements
-
-{list here sources of all reused/adapted ideas, code, documentation, and third-party libraries -- include links to the original source as well}
+## Content Page
+
+* [Introduction](#introduction)
```
[the remark this answers is not in the document]
Comment (SeanHoWB) [reply docs/DeveloperGuide.md:5]: sure man, if merge alr then we work on it?

File: docs/DeveloperGuide.md (hunk 4 above)
Comment (SeanHoWB) [reply docs/DeveloperGuide.md:61]: thanks! resolved

**Comment by limjierui** [comment]
Nicely done! Looks good to go!

**Pull request merged**